debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnector.java

```diff
@@ -82,9 +82,9 @@ public Config validate(Map<String, String> connectorConfigs) {
         ConfigValue hostnameValue = results.get(MySqlConnectorConfig.HOSTNAME.name());
         ConfigValue portValue = results.get(MySqlConnectorConfig.PORT.name());
         ConfigValue userValue = results.get(MySqlConnectorConfig.USER.name());
-        ConfigValue passwordValue = results.get(MySqlConnectorConfig.PASSWORD.name());
+        final String passwordValue = config.getString(MySqlConnectorConfig.PASSWORD);
 
-        if (passwordValue.value() == null || ((String)passwordValue.value()).isEmpty()) {
+        if (passwordValue == null || passwordValue.isEmpty()) {
```

Comment:
Could use `Strings#nullOrEmpty()`, too.

Reply:
I'll merge it as is; if you like, simply adjust it and push the commit yourself. Thanks!